### PR TITLE
August newsletter

### PR DESCRIPTION
**Pull request by bethaitman**
Now ready for review!

### CODE REVIEW

File: docs/blog/newsletter-august-2019.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+- It can be reasonable to ask a candidate to complete a writing test before interviews, so that you get an indication of their skills. Just be careful about how much time it will take them.
+- An alternative is an on-site writing test during an interview day. But keep the test to an hour or less, and be aware that time pressure changes how polished you can expect the work to be.
+- Be wary of writing tests on real company topics: while these are more realistic tests, they can also look like you're asking for free work. Instead, perhaps ask them to document a tool or open-source project with no direct benefit to the company.
+- Getting candidates to think about the audience of the documentation is crucial!
+- You can learn a lot about how a person works, so give candidates a chance to talk about their writing process after the test. 
+
+For ideas, read Cockroach Labs' `Documentation Exercises <https://github.com/cockroachlabs/open-sourced-interview-process/blob/master/DocumentationExercises.md>`__.
+
```
Comment (Bradamant3) [inline docs/blog/newsletter-august-2019.rst:46]: Shoutout to Kyla for a beautiful writeup of a topic that makes me seeth. Almost I'm convinced of the value of writing tests even!

**Review by Bradamant3 — APPROVED** [review]
LGTM. Approved -- but someone else should probably take a look too.

File: docs/blog/newsletter-august-2019.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+Write the Docs Newsletter – August 2019
+#######################################
+
+Hello everyone! It's Beth and the team here, with another packed newsletter issue for you.
+
+A quick correction first: last month we said that the Australia conference is in Melbourne, when it is in fact in **Sydney**. Really sorry about that. If you'd like to speak in **Sydney**, the `CFP </conf/australia/2019/cfp/>`__ is still open: you have until 9th August to submit.
+
+In the other hemisphere, the `Prague speaker speaker schedule is up </conf/prague/2019/schedule/>`__, and `tickets are almost sold out </conf/prague/2019/news/events-activities/>`__. Once they're gone, they're gone, so if you'd like to come, buy your tickets ASAP!
+
+One last thing - we're delighted to have some new recruits to the newsletter team, which means we now have one more story about what the community has been talking about on Slack! Enjoy :) The newsletter team takes September off, so we'll be back in your inboxes come October.
```
Comment (plaindocs) [inline docs/blog/newsletter-august-2019.rst]: @ericholscher did you want (and/or want to mention the community update here?)

File: docs/blog/newsletter-august-2019.rst [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+------------------------------------------------
+Adventures in generating docs from code comments
+------------------------------------------------
+
+A question about how to improve on tools like `Javadoc <https://www.oracle.com/technetwork/java/javase/documentation/index-jsp-135444.html>`__ and `doxygen <http://www.doxygen.nl/>`__ yielded many insights into getting great results with docs generated from code comments:
+
+* Javadoc is pretty standard in the Java world, so you can expect that Java devs are already looking for the Javadoc - and you can help them just by making sure it's easy to find.
```
Comment (plaindocs) [inline docs/blog/newsletter-august-2019.rst]: That is a lot of java on one line!

File: docs/blog/newsletter-august-2019.rst [hunk 4]
```diff
@@ -22,17 +20,17 @@ Adventures in generating docs from code comments
...
+* Javadoc interprets the code itself, so it can - even without code comments - generate basic docs on package, class, method, field, and argument names and types. One person mentioned they sometimes run Javadoc just to see what it turns up about the code that isn't documented elsewhere.
 
-For more about a successful Javadoc/doxygen project that included writers and developers, check out Jennifer Rondeau's `Symantec case study <https://www.docslikecode.com/articles/symantec-case-study/>`__. For a discussion of the benefits of writers and developers working together, see Beth Aitman's talk `Who Writes the Docs? </videos/portland/2018/who-writes-the-docs-beth-aitman/>`__.
+For more about a successful Javadoc/doxygen project that included writers and developers, check out Jennifer Rondeau's `Symantec case study <https://www.docslikecode.com/articles/symantec-case-study/>`__. For a discussion of the benefits of writers and developers working together, see our editor's talk `Who Writes the Docs? </videos/portland/2018/who-writes-the-docs-beth-aitman/>`__.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-august-2019.rst]: I didn't notice this until I looked at the preview, but the period after the link setup for Who Writes the Docs? should be removed:
...talk `Who Writes the Docs? </videos/portland/2018/who-writes-the-docs-beth-aitman/>`__

**Review by plaindocs — APPROVED** [review]
:100:

**Review by hillaryfraley — APPROVED** [review]
Hey, just the one suggestion about removing a period, and otherwise LGTM! I proofread and clicked the links and didn't find anything.

**Review by clairealyssa — APPROVED** [review]
I didn't see anything that hadn't already been seen. Excellent job, as always, Beth!

File: docs/blog/newsletter-august-2019.rst (hunk 4 above)
Comment (bethaitman) [reply docs/blog/newsletter-august-2019.rst]: Good spot, thank you!

File: docs/blog/newsletter-august-2019.rst (hunk 2 above)
Comment (bethaitman) [reply docs/blog/newsletter-august-2019.rst]: The community update? I don't know what that refers to

File: docs/blog/newsletter-august-2019.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-august-2019.rst]: I already removed a few :D will try to get it down a bit more!

Comment (hillaryfraley) [reply docs/blog/newsletter-august-2019.rst]: :D I guess I did overdo it

Comment (bethaitman) [reply docs/blog/newsletter-august-2019.rst]: It's super difficult with this topic!

**Comment by bethaitman** [comment]
Awesome, thank you all. Will merge tomorrow am barring the question that Sam had for @ericholscher about something extra to include?

**Pull request merged**